_posts/2022-12-29-vanilla-os-22-10-kinetic.md

```diff
@@ -62,11 +62,9 @@ system.
 
 ![Apx Control Center](/assets/uploads/vanilla-os-apx-control-center.png)
 
-For GUI packages created inside apx containers, .desktop files are created 
-automatically and added to the Applications menu. These applications are also 
-displayed beside other applications in the “Open with” menu in nautilus. 
-GUI Packages installed inside containers get shown in the Sub System section 
-of Vanilla control center.
+Graphical applications installed inside apx containers are automatically added 
+to the Applications menu as well as the “Open with” menu in the file manager. 
```

Comment:
```suggestion
to the Applications menu as well as the “Open with” menu in the file manager (`nautilus`). 
```

Reply:
I guess you really like nautilus 😛 There's really no reason to clutter the sentence with this. The name 'nautilus' is only ever used as a codename, it's never exposed to the user.

Reply:
> I guess you really like nautilus 😛 There's really no reason to clutter the sentence with this. The name 'nautilus' is only ever used as a codename, it's never exposed to the user. 

Agreed, I will mark this as resolved 🙂